src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -1226,4 +1226,12 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
       return 0;
     }
   }
+
+  getDrmInfo(): ?PKDrmDataObject {
+    if (NativeAdapter._drmProtocol && this._drmHandler) {
```

Comment:
why the previous way was wrong? (`_sourceObj.drmData`)?

Reply:
_sourceObj.drmData is array of configured drm data objects , but is not indicating the actual active drm (like _drmProtocol dose)